### PR TITLE
zh-CN: cr2 Android translations

### PR DESCRIPTION
**Pull request by henrif75**
*Stats:*
Before: 1018 translated messages, 368 fuzzy translations, 2074 untranslated messages.
After: 1132 translated messages, 456 fuzzy translations, 1872 untranslated messages.
#324 #1463

### CODE REVIEW

File: po/zh-CN.po [hunk 1]
```diff
@@ -11365,16 +11394,16 @@ msgstr "_birthday_service/src/lib.rs_:"
...
 #: src/android/aidl/implementation.md:8
 msgid "//! Implementation of the `IBirthdayService` AIDL interface.\n"
-msgstr ""
+msgstr "//! Implementation of the `IBirthdayService` AIDL interface.\n"
 
 #: src/android/aidl/implementation.md:11
 #, fuzzy
 msgid "/// The `IBirthdayService` implementation.\n"
-msgstr "服务实现"
+msgstr "/// HelloWorld::hello method implementation.\n"
```
Comment (emmali01) [inline po/zh-CN.po:11402]:
```suggestion
msgstr "`IBirthdayService`实现 "
```

File: po/zh-CN.po [hunk 2]
```diff
@@ -11991,35 +12038,37 @@ msgstr ""
 #: src/chromium/interoperability-with-cpp/error-handling-png.md:9
 #, fuzzy
 msgid "\"Rust\""
-msgstr "Rustdoc"
+msgstr "\"rust\""
```
Comment (emmali01) [inline po/zh-CN.po:12041]:
```suggestion
msgstr "\"Rust\""
```

File: po/zh-CN.po [hunk 3]
```diff
@@ -12112,15 +12169,19 @@ msgid ""
...
 msgstr ""
+"共享类型的 `#[derive()]` 支持有限数量的 trait。系统还会针对 C++ 代码生成相应"
+"的功能，例如，如果您派生了 `Hash`，还会为相应的 C++ 类型生成 `std::hash` 实"
+"现。"
 
 #: src/android/interoperability/cpp/shared-enums.md:15
 #, fuzzy
 msgid "Generated Rust:"
-msgstr "不安全 Rust"
+msgstr "生成的 C++："
```
Comment (emmali01) [inline po/zh-CN.po:12179]:
```suggestion
msgstr "生成的 Rust："
```

File: po/zh-CN.po [hunk 4]
```diff
@@ -12129,47 +12190,53 @@ msgid ""
...
 msgid "\"Success!\""
-msgstr ""
+msgstr "\"Success!\""
 
 #: src/android/interoperability/cpp/rust-result.md:22
 msgid ""
 "Rust functions that return `Result` are translated to exceptions on the C++ "
 "side."
-msgstr ""
+msgstr "在 C++ 方面，返回 `Result` 的 Rust 函数会被视为异常。"
```
Comment (emmali01) [inline po/zh-CN.po:12208]:
```suggestion
msgstr "在 C++ 方面，返回 `Result` 的 Rust 函数会被翻译为异常。"
```

File: po/zh-CN.po [hunk 5]
```diff
@@ -12192,7 +12262,7 @@ msgstr "C++ 示例"
 #: src/android/interoperability/cpp/type-mapping.md:5
 #, fuzzy
 msgid "`rust::String`"
-msgstr "String"
+msgstr "`rust_bindgen`"
```
Comment (emmali01) [inline po/zh-CN.po:12265]:
```suggestion
msgstr "`rust::String`"
```

File: po/zh-CN.po [hunk 6]
```diff
@@ -12201,7 +12271,7 @@ msgstr "`&str`"
 #: src/android/interoperability/cpp/type-mapping.md:6
 #, fuzzy
 msgid "`rust::Str`"
-msgstr "`rust_test`"
+msgstr "`rust::Box<T>`"
```
Comment (emmali01) [inline po/zh-CN.po:12274]:
```suggestion
msgstr "`rust::Str`"
```

File: po/zh-CN.po [hunk 7]
```diff
@@ -12242,7 +12314,7 @@ msgstr "`Vec`"
 #: src/android/interoperability/cpp/type-mapping.md:11
 #, fuzzy
 msgid "`rust::Vec<T>`"
-msgstr "`mpsc::Receiver<T>`"
+msgstr "`rust::Box<T>`"
```
Comment (emmali01) [inline po/zh-CN.po:12317]:
```suggestion
msgstr "`rust::Vec<T>`"
```

File: po/zh-CN.po [hunk 8]
```diff
@@ -12252,36 +12324,40 @@ msgstr "`Cell<T>`"
 #: src/android/interoperability/cpp/type-mapping.md:12
 #, fuzzy
 msgid "`std::vector<T>`"
-msgstr "`mpsc::Receiver<T>`"
+msgstr "`std::unique_ptr<T>`"
```
Comment (emmali01) [inline po/zh-CN.po:12327]:
```suggestion
msgstr "`std::vector<T>`"
```

File: po/zh-CN.po [hunk 9]
```diff
@@ -12476,18 +12574,19 @@ msgid "_interoperability/java/HelloWorld.java_:"
...
+#, fuzzy
 msgid "\"helloworld_jni\""
-msgstr ""
+msgstr "\"helloworld_jni\""
 
 #: src/android/interoperability/java.md:68
 #, fuzzy
 msgid "\"HelloWorld.java\""
-msgstr "Hello World!"
+msgstr "\"helloworld_jni\""
```
Comment (emmali01) [inline po/zh-CN.po:12584]:
```suggestion
msgstr "\"HelloWorld.java\""
```

File: po/zh-CN.po [hunk 10]
```diff
@@ -12476,18 +12574,19 @@ msgid "_interoperability/java/HelloWorld.java_:"
...
 
 #: src/android/interoperability/java.md:69
 #, fuzzy
 msgid "\"HelloWorld\""
-msgstr "Hello World!"
+msgstr "\"helloworld_jni\""
```
Comment (emmali01) [inline po/zh-CN.po:12589]:
```suggestion
msgstr "Hello World!"
```

**Review by emmali01 — APPROVED** [review]

**Review by SketchK — APPROVED** [review]

**Comment by henrif75** [comment]
Thanks @emmali01 and @SketchK !

**Pull request merged**